### DISCUSSION TITLE
What is the GPS button on the mainscreen for?

### DISCUSSION THREAD

**Discussion by Matombo** [OpenTracksApp/OpenTracks#900] [Q&A] [answered]
On the main screen between the menu and the statistic button is a GPS button.
What is that actually for?

**Comment by dennisguse** [comment, marked as answer]
It can be used to tell the phone already to start the GPS and get a fix before actually starting a track.
Especially some older phones need some time (best case one is standing still) to acquire a GPS fix and then you are sure that the track is actually starting were you started and not at some later location.

**Reply by Matombo** [reply]
Thanks for the clarification.
And thanks for maintaining this neat little app ^^.
